lib/SubscriptionItem.php

```diff
@@ -64,16 +65,16 @@ public function usageRecordSummaries($params = null, $opts = null)
     }
 
     /**
-     * @param string $id
+     * @param string $id The ID of the subscription item on which to retrieve the usage record summaries.
      * @param array|null $params
      * @param array|string|null $opts
      *
      * @throws \Stripe\Exception\ApiErrorException if the request fails
      *
-     * @return Collection The list of usage record summaries.
+     * @return \Stripe\Collection The list of usage record summaries.
```

Comment:
Slightly more than what was on the label (and the constant being factored out) but seems peaceful

Reply:
:shame: You got me :)